src/interactions/services/comment_reactions.py

```diff
@@ -24,6 +24,16 @@ def react_to_comment(
     return reaction
 
 
+def get_comment_reactions(comment: Comment) -> dict | None:
+
+    reactions = CommentReaction.objects.filter(comment=comment)
+
+    if reactions:
+        return reactions
+
+    return None
```

Comment:
why not just 
```suggestion
    return CommentReaction.objects.filter(comment=comment)
```

it means you'd get `[]` instead of `None` for none, but that's easier to e.g. iterate over etc and fits the QS return type (which I think is nicer as more expected).

This way you're also forcing the execution of the QS, which is lazy - which may or may not be intentional but feels a bit early given this is a service function...